README.md

```diff
@@ -2,32 +2,32 @@ Rancher Harvester
 ========
 [![Build Status](https://drone-publish.rancher.io/api/badges/rancher/harvester/status.svg)](https://drone-publish.rancher.io/rancher/harvester)
 
-Rancher Harvester is an open source Hyper-Converged Infrastructure(HCI) solution based on Kubernetes.
+Rancher Harvester is an open source Hyper-Converged Infrastructure(HCI) solution based on Kubernetes. It is an on-prem cloud computing virtualization platform.
```

Comment:
https://en.wikipedia.org/wiki/Hyper-converged_infrastructure has a bit more description of HCI. 

Btw, the style seems like `Hyper-converged infrastructure` (Nutanix uses this) or `hyperconverged infrastructure`(VMWare uses this).

Reply:
I think hyper-converged infrastructure is better because it seems that "hyperconverged" isn't considered to be a dictionary word everywhere yet.